src/components/ec-blog-item/ec-blog-item.story.js

```diff
@@ -0,0 +1,41 @@
+import { storiesOf } from '@storybook/vue';
+import EcBlogItem from './ec-blog-item.vue';
+
+const stories = storiesOf('Blog Item', module);
```

Comment:
We don't use storiesOf since the v6 it's active. Take a look [here](https://fxsolutions.atlassian.net/browse/ONL-4998)